packages/react-vapor/src/components/refresh/RefreshCallback.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {IReactVaporState} from '../../ReactVapor';
+import {callIfDefined} from '../../utils/FalsyValuesUtils';
+import {IDispatch} from '../../utils/ReduxUtils';
+import {RefreshCallBackActions} from './RefeshCallbackActions';
+import {RefreshStatus, RefreshStatusSelectors} from './RefreshCallbackReducer';
+
+export interface IRefreshCallbackOwnProps {
+    id: string;
+    time?: number;
+    incremental?: number;
```

Comment:
What is the usage of incremental?

Reply:
removed. set the number you want to increment every time your interval reset ( 1000ms )